services/brig/src/Brig/Data/Blacklist.hs

```diff
@@ -27,3 +40,41 @@ keySelect = "SELECT key FROM blacklist WHERE key = ?"
 
 keyDelete :: PrepQuery W (Identity Text) ()
 keyDelete = "DELETE FROM blacklist WHERE key = ?"
+
+--------------------------------------------------------------------------------
+-- Excluded phone prefixes
+
+insertPrefix :: MonadClient m => PhonePrefix -> m ()
+insertPrefix prefix = retry x5 $ write prefixInsert (params Quorum (Identity $ fromPhonePrefix prefix))
+
+deletePrefix :: MonadClient m => PhonePrefix -> m ()
+deletePrefix prefix = retry x5 $ write prefixDelete (params Quorum (Identity $ fromPhonePrefix prefix))
+
+existsPrefix :: MonadClient m => PhonePrefix -> m Bool
+existsPrefix prefix = return . isJust =<< fmap runIdentity <$>
```

Comment:
Not sure `fmap runIdentity` is needed here

Reply:
Also, `return . isJust =<<` == `isJust <$>`

Reply:
thanks. Actually, this function ended up not being used, so it's removed entirely.